test/sanity/validate-modules/ignore.txt

```diff
@@ -506,10 +506,10 @@ lib/ansible/modules/cloud/profitbricks/profitbricks.py E324
 lib/ansible/modules/cloud/profitbricks/profitbricks.py E326
 lib/ansible/modules/cloud/profitbricks/profitbricks_datacenter.py E326
 lib/ansible/modules/cloud/profitbricks/profitbricks_nic.py E324
+lib/ansible/modules/cloud/profitbricks/profitbricks_nic.py E325
```

Comment:
New ignores should not be added. The underlying issue should be fixed instead. If the issue is in common code (module_utils or docs fragments), those issues should be fixed in a separate PR first.

Reply:
Resolved, thanks @mattclay